proto/v1alpha1/swagger/grafeas.swagger.json

```diff
@@ -226,41 +229,97 @@
         },
         "parameters": [
           {
-            "name": "name",
-            "description": "The name field will contain the note name for example:\n  \"provider/{provider_id}/notes/{note_id}\"",
+            "name": "name_2",
+            "description": "The name of the project in the form of `projects/{PROJECT_ID}`.",
             "in": "path",
             "required": true,
-            "type": "string"
+            "type": "string",
+            "pattern": "projects/[^/]+"
+          }
+        ],
+        "tags": [
+          "GrafeasProjects"
+        ]
+      },
+      "delete": {
+        "summary": "Deletes the specified project.",
+        "operationId": "GrafeasProjects_DeleteProject",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "properties": {}
+            }
           },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/rpcStatus"
+            }
+          }
+        },
+        "parameters": [
           {
-            "name": "filter",
-            "description": "The filter expression.",
-            "in": "query",
-            "required": false,
-            "type": "string"
+            "name": "name_2",
+            "description": "The name of the project in the form of `projects/{PROJECT_ID}`.",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "projects/[^/]+"
+          }
+        ],
+        "tags": [
+          "GrafeasProjects"
+        ]
+      },
```

Comment:
UpdateOperation is now grouped with the Project methods, this is due to the way grpc-gateway generates swagger "greedily" from google.api.http annotations.